Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -130,12 +152,16 @@ final class PledgePaymentMethodsViewController: UIViewController {
     self.viewModel.inputs.configureWith(pledgePaymentMethodsValue)
   }
 
-  // MARK: - Actions
+  // MARK: - Accessors
 
   @objc private func applePayButtonTapped() {
     self.viewModel.inputs.applePayButtonTapped()
   }
 
+  func updatePledgeButton(_ enabled: Bool) {
+    self.viewModel.inputs.updatePledgeButtonEnabled(isEnabled: enabled)
```

Comment:
The real "brains" of the pledge button enabled state is actually the `PledgeViewModel`. This is because we expect the pledge button's enabled state to be driven by a variety of factors: whether the reward minimum is met, whether a shipping location has been selected, etc. Since the `PledgeViewModel` is the only source of truth for this information, it needs to be the one to drive the pledge button's enabled state.